Sources/Onboarding/Callouts/OnboardingBottomBarView.swift

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveUI
+import BraveShared
+
+public struct OnboardingBottomBarView: View {
```

Comment:
Once we know more about v2, we can expand this view to be more generalized. Nothing for now